src/pandas_profiling/report/structure/correlations.py

```diff
@@ -68,7 +68,9 @@ def get_correlation_items(summary) -> Optional[Renderable]:
         vmin, name, description = key_to_data[key]
 
         diagram = Image(
```

Comment:
The rendering logic for the table should be here (and not in plot.py, since we don't generate a plot).